numpyro/infer/hmc.py

```diff
@@ -412,11 +411,6 @@ def _init_state(self, rng_key, model_args, model_kwargs, init_params):
                 init_strategy=self._init_strategy,
                 model_args=model_args,
                 model_kwargs=model_kwargs)
```

Comment:
why is this being removed?

Reply:
This is a deprecation message. Previously, we treat `param` sites are improper latent variables but that support has been removed from 0.2 release IIRC. Now, for parameters of BNN, we still hold their shape information in `param` sites. Displaying this message is unnecessary for that usage case.